cmd/build/command.go

```diff
@@ -181,6 +181,13 @@ func (bc *Command) getBuilder(options *types.BuildOptions, okCtx *okteto.Context
 				bc.analyticsTracker.TrackImageBuild,
 				bc.insights.TrackImageBuild,
 			}
+			if !okCtx.IsOktetoCluster() && manifest.Type == model.StackType {
+				return nil, oktetoErrors.UserError{
+					E: fmt.Errorf("docker Compose format is only available using the Okteto Platform"),
+					Hint: `Follow this link to install the Okteto Platform in your Kubernetes cluster:
+    https://www.okteto.com/docs/get-started/install`,
+				}
+			}
```

Comment:
Build command is not supported for non-okteto contexts, so we don't arrive to this point, so we don't show this message. This is what I get with this CLI version

```
ok build -f docker-compose.yml
 i  Using okteto @ xxxxxxxx as context
 x  This command is only available installing Okteto in your cluster
    Visit our docs to learn more about the Okteto Platform:
    https://www.okteto.com/docs
```